src/choices.c

```diff
@@ -14,7 +14,7 @@
 #include "choices.h"
 
 size_t
-min_match_length(char *str, char *query)
+min_match_length(char *str, char *query, int *start_pos)
```

Comment:
I don't have a preference on the naming but the implementation would change in that case, since right now this still returns the same length but then the index of the first character of the match, not whether or not there was one.